truffle.js

```diff
@@ -2,8 +2,12 @@ const assert = require('assert')
 const HDWalletProvider = require('truffle-hdwallet-provider')
 const DEFAULT_GAS_PRICE_GWEI = 5
 const GAS_LIMIT = 6.5e6
+
 const DEFAULT_MNEMONIC = 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'
 
+// Load env vars
+require('dotenv').load()
```

Comment:
Don't use `.load` use `.config`. Load is deprecated and still there for backwards compatibility.